### ISSUE TITLE
Job atac.read_genome_tsv:NA:1 exited with return code -1 which has not been declared as a valid return code. 

### ISSUE THREAD

**Issue by zfrenchee** [ENCODE-DCC/atac-seq-pipeline#92]
**Describe the bug**
I'm new to cromwell, and hoping to run this pipeline on our data.  `read_genome_tsv` is failing, even though the command is as simple as 

```
	command {
		cat ${genome_tsv} > 'tmp.tsv'
	}
```
Less importantly, it's also giving me an unhelpful error message, because of a strange exit code number. 

**OS/Platform and dependencies**
- Debian 8.11
- Cromwell 36.1, via docker image (`broadinstitute/cromwell:36-858f647`) in server mode.
- local backend, trying to run this pipeline via docker: workflowOptions: 
```
    "default_runtime_attributes" : {
        "docker" : "quay.io/encode-dcc/atac-seq-pipeline:v1.1.6",
```

**Attach error logs**
From /metadata endpoint:
```
        "executionStatus": "Failed",
        "failures": [
          {
            "causedBy": [],
            "message": "Job atac.read_genome_tsv:NA:1 exited with return code -1 which has not been declared as a valid return code. See 'continueOnReturnCode' runtime attribute for more details."
          }
        ],
```

Cromwell Return Code Spec: https://github.com/broadinstitute/cromwell/blob/develop/backend/src/test/scala/cromwell/backend/validation/ContinueOnReturnCodeSpec.scala

My cromwell produces no stderr logs. Why not? This file/directory doesn't exist: 
```
        "stderr": "/cromwell-working-dir/cromwell-executions/atac/45c7b99d-486e-4db3-b619-0ae1c2adf948/call-read_genome_tsv/execution/stderr",
```
In fact, from `cromwell-executions/`  the command 
```
find . -type f -name 'stdout' -or -name 'stderr'
``` 
finds nothing. 

**TL;DR:**
- Why is `read_genome_tsv` failing? 
- Why do I get no stderr logs? 
- Consider fixing `read_genome_tsv` exit code, in case that means a more helpful error message.

**Comment by leepc12** [comment]
First of all cromwell 35/36 is buggy. Please try with cromwell-34 or 38.
Please check if you specified a correct genome TSV file in your input JSON. Check if TSV exists and all paths in it are valid.

I guess that cromwell generated stdout/stderr inside the container and for some strange reasons it failed to move them outside the container.

**Comment by zfrenchee** [comment]
@leepc12 thanks for your reply. 
- The paths inside the container are definitely valid
- The stdout/stderr files don't exist inside the container either. 

I'll try with 34.

**Comment by zfrenchee** [comment]
@leepc12 also didn't work in cromwell 38 docker container. Fixed in cromwell 34 jar.
